examples/c#/AnnotateDocumentExample/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Grpc.Core;
+using Grpc.Net.Client;
+using Ssn.Annotator.V1;
+using Confidence = Ssn.Type.Confidence;
+using Document = Ssn.Annotator.V1.Document;
+
+namespace ConsoleApp1;
+
+static class Program {
+    public static void Main(string[] args) {
+        annotateDocument();
+    }
+
+    static void annotateDocument()
+    {
+        //channel
+        var chnl = GrpcChannel.ForAddress("https://api.snbx.ssn.visma.ai:443");
+        //client
+        var clnt = new DocumentAnnotator.DocumentAnnotatorClient(chnl);
+        //request https://www.africau.edu/images/default/sample.pdf
+        var rqst = new DocumentAnnotatorRequest
+        {
+            Document = new Document
+            {
+                Source = new DocumentSource
+                {
+                    HttpUri = "https://www.africau.edu/images/default/sample.pdf"
```

Comment:
Let's try and use an actual invoice so people don't get horrified when they see that Smartscan is very sure about values which are not even present on the current pdf 😄 Smartscan is smart but it's mostly invoice-receipt-smart! I'd suggest adding [this file](https://visma.slack.com/archives/CG4QZH3EH/p1674810189828289?thread_ts=1674740070.189149&cid=CG4QZH3EH) to the directory and sending that instead to the annotate endpoint.